pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -147,6 +155,72 @@ func GatherMostRecentMetrics(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+// GatherContainerImages collects essential information about running containers.
+// Specifically, the age of pods, the set of running images and the container names are collected.
+//
+// Location in archive: config/running_containers.json
+func GatherContainerImages(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		records := []record.Record{}
+
+		contInfo := ContainerInfo{
+			Images:     ContainerImageSet{},
+			Containers: PodsWithAge{},
+		}
+		// Cache for image indices in the collected set of container image.
+		image2idx := map[string]int{}
+
+		// Use the Limit and Continue fields to request the pod information in chunks.
+		continueValue := ""
+		for {
+			pods, err := i.coreClient.Pods("").List(metav1.ListOptions{
+				Limit:    imageGatherPodLimit,
+				Continue: continueValue,
+				// FieldSelector: "status.phase=Running",
+			})
+			if err != nil {
+				return nil, []error{err}
+			}
+
+			for podIndex, pod := range pods.Items {
+				podPtr := &pods.Items[podIndex]
+				if strings.HasPrefix(pod.Namespace, "openshift") && hasContainerInCrashloop(podPtr) {
+					records = append(records, record.Record{Name: fmt.Sprintf("config/pod/%s/%s", pod.Namespace, pod.Name), Item: PodAnonymizer{podPtr}})
+				} else if pod.Status.Phase == corev1.PodRunning {
+					for _, container := range pod.Spec.Containers {
+						imageURL := container.Image
```

Comment:
You really should not be using this value, you should be using the status details because that is the image that was actually loaded.  And you should not be parsing image urls this way, you should be using teh image reference code in `github.com/openshift/library-go/pkg/image/reference` which is safe to parse.